### PR TITLE
Converting to masked layers bug fix

### PR DESCRIPTION
**Pull request by yc7z**
# PR Type
Fix

# Short Description

Fixed the bug for converting a model to the masked version by setting the mask scores parameters at the appropriate module level.

# Tests Added

Added a new test that uses `load_state_dict`.

### CODE REVIEW

**Review by emersodb — APPROVED** [review]
Changes look good to me...very subtle bug. Good catch!
